packages/containers/src/SelectObject/SelectObject.component.js

```diff
@@ -16,25 +17,49 @@ function SelectObject(props) {
 			...props.schema,
 		};
 	}
+	const { className = '', ...filter } = props.filter || {};
 	return (
 		<div className={`tc-select-object ${theme.wrapper}`}>
-			<FilterBar id={props.id} dockable={false} navbar={false} {...props.filter} />
+			<FilterBar
+				className={classNames(className, theme.filter)}
+				id={props.id}
+				dockable={false}
+				navbar={false}
+				{...filter}
+			/>
 			<div className={theme.container}>
-				{!props.tree ? (
-					<List
-						{...props.list}
-						id={`${props.id}-list`}
-						data={props.filteredData}
-						className={theme.list}
-					/>
-				) : (
-					<TreeView
-						{...props.tree}
-						componentId={props.id}
-						noHeader
-						data={props.filteredData}
-						className={theme.tree}
-					/>
+				{!props.tree &&
+					!props.filteredData && (
+						<List
+							{...props.list}
+							id={`${props.id}-list`}
+							data={props.sourceData}
+							className={theme.list}
+						/>
+					)}
+				{props.tree &&
+					!props.filteredData && (
+						<TreeView
+							{...props.tree}
+							componentId={props.id}
+							noHeader
+							data={props.sourceData}
+							className={theme.tree}
+						/>
+					)}
+				{props.filteredData && (
+					<ListGroup className={theme.results}>
+						{props.filteredData.map(data => (
+							<ListGroupItem
+								key={data.get(props.idAttr)}
+								header={data.get(props.nameAttr)}
+								onClick={event => props.results.onClick(event, data)}
+								active={props.results.selectedId === data.get(props.idAttr)}
+							>
+								{data.get('currentPosition')}
```

Comment:
You get id/name attribute name from props but this one is hard coded

Reply:
yes it's because it's injected by the container, so it's not a special data.
id, name and children was given by the data itself